src/Upgrade/Infrastructure/Configuration/ConfigurationProvider.php

```diff
@@ -24,6 +24,16 @@ class ConfigurationProvider
      */
     public const GITHUB_SOURCE_CODE_PROVIDER = 'github';
 
+    /**
```

Comment:
Can you please add a specification annotation with a short description per each method?

Reply:
Yes ++